op-node/flags/flags.go

```diff
@@ -84,33 +101,39 @@ var (
 			out := sync.CLSync
 			return &out
 		}(),
+		Category: RollupCategory,
 	}
 	RPCListenAddr = &cli.StringFlag{
-		Name:    "rpc.addr",
-		Usage:   "RPC listening address",
-		EnvVars: prefixEnvVars("RPC_ADDR"),
-		Value:   "127.0.0.1",
+		Name:     "rpc.addr",
+		Usage:    "RPC listening address",
+		EnvVars:  prefixEnvVars("RPC_ADDR"),
+		Value:    "127.0.0.1",
+		Category: OperationsCategory,
```

Comment:
It may be confusing that the `P2PCategory` actually has the label  `"NETWORKING"`, and network configuration like these rpc flags are under the `OperationsCategory`.
Why didn't you name the p2p category something like `PEER-2-PEER`, and add stuff like rpc config into a separete `NETWORKING` category?

Reply:
Can rename `Networking` to `P2P`. the RPC stuff is not `networking` imo. I copied geth here, but we do already have the notion of P2P so maybe that is cleaner.

Reply:
sgtm